src/index.html

```diff
@@ -4,11 +4,32 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpackdede Exercise</title>
+    <title>LeaderBoard</title>
 </head>
 
 <body>
-    <h1>Hello w!</h1>
+    <h1>Leaderboard</h1>
+    <div class="container">
+        <section class="list">
+            <div class="refresh">
+                <h2>Recent scores</h2>
+                <button>Refresh</button>
+            </div>
+            <ul>
+                <li>name:100</li>
+                <li>name:50</li>
+                <li>name:20</li>
+            </ul>
+        </section>
+        <section class="form">
+            <h2>Add your score</h2>
+            <form>
+                <input type="text" id="username" required />
+                <input type="number" id="score" required />
```

Comment:
- [ ] I think it would be more comfortable to use for users if we add placeholder text to input fields. What do you think?

Reply:
Sure I will do it 👌